client/app/pages/queries/source-view.js

```diff
@@ -96,9 +96,10 @@ function QuerySourceCtrl(
         },
       })
       .result.then((param) => {
-        $rootScope.$broadcast('query-editor.command', 'paste', '{{ ' + param.name + ' }}');
-        $rootScope.$broadcast('query-editor.command', 'focus');
         $scope.query.getParameters().add(param);
+        param = $scope.query.getParameters().get(param.name);
```

Comment:
Is it different than what this function gets as an argument?

Reply:
Yes. An argument is just a plain object, but here I need a `Parameter` instance, and `.getParameters().get(name)` returns it

Reply:
@arikfr I changed this fragment a bit: I decided that adding an argument to `.getParameters().get()` just to use it in a single place wasn't a good idea, so `.getParameters().add(...)` now returns newly added `Parameter` instance.